jest.config.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
-  testEnvironment: 'node',
+  preset: '@siroc/jest-preset',
```

Comment:
If @siroc/* are implicitly installed alongside with siroc, what is the benefit of package spitting? This would also make supporting yarn 2 harder as it requires directly adding them as deps and if directly adding, module authors should update multiple dev dependencies.

Reply:
The thought was that the 1-click easy install was just one package, but a more finely-grained installation could be achieved by just installing parts. No functionality in `siroc` package, just bundling.

Reply:
I can add `@siroc/jest-preset` to `package.json` to address the potential yarn issue?